_inc/client/traffic/ads.jsx

```diff
@@ -31,6 +31,10 @@ export const Ads = moduleSettingsForm( class extends React.Component {
 		analytics.tracks.recordJetpackClick( 'view-earnings' );
 	};
 
+	trackConfigureWidgetClick = () => {
+		analytics.tracks.recordJetpackClick( 'place-ad-widget' );
```

Comment:
@withinboredom should it be a general rule that props should not use dashes? Or was #9277 a specific thing?

Reply:
anything that doesn't follow the naming conventions outlined in the tracks naming convention page on the fieldguide will be rejected. no dashes, underscores only: `<source>_<context>_<optional-subcontext>_<action>_<optional-qualifier>`

Reply:
proposed fixing this one in https://github.com/Automattic/jetpack/pull/9404

There are a lot of `analytics.tracks.recordJetpackClick` in jetpack that use hyphens. Are these ignored by tracks because they don't follow the convention? If so, can we switch them out without causing any other issues?